src/main/java/com/kodilla/ecommercee/domain/ProductDto.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee.domain;
```

Comment:
powinno byc w pakiecie com.kodilla.ecommercee.dto jak juz, ale po co to do implementacji encji?